doc/book/admin/access_control.rst

```diff
@@ -1,377 +1,1069 @@
+
+    box.schema.user.grant(username, privileges, object-type, object-name[, {options}])
+
+*   ``username``: the name of the user that gets the specified privileges.
+*   ``privileges``: a string value that represents :ref:`privileges <access_control_list_privileges>` granted to the user. If there are several privileges, they should be separated by commas without a space.
+*   ``object-type``: a type of :ref:`object <access_control_list_objects>` to which privileges are granted.
+*   ``object-name``: the name of the object to which privileges are granted.
+    An empty string (``""``) or ``nil`` provided instead of ``object-name`` grants the specified privileges to all objects of the specified type.
+
+    ..  NOTE::
+
+        ``object-name`` is ignored for the following combinations of privileges and object types:
+
+        *   Any privilege granted to ``universe``.
+        *   The ``create`` and ``drop`` privileges for the following object types: ``user``, ``role``, ``space``, ``function``, ``sequence``.
+        *   The ``execute`` privilege for the following object types: ``lua_eval``, ``lua_call``, ``sql``.
+
+
+.. _access_control_grant_creating_any_obj:
+
+Any object
+~~~~~~~~~~
+
+In the example below, ``testuser`` gets privileges allowing them to create any object of any type:
+
+..  code-block:: lua
+
+    box.schema.user.grant('testuser','read,write,create','universe')
+
+In this example, ``testuser`` can grant access to objects that ``testuser`` created:
+
+..  code-block:: lua
+
+    box.schema.user.grant('testuser','write','space','_priv')
+
+
+
+
+.. _access_control_grant_spaces:
+
+Spaces
+~~~~~~
+
+.. _access_control_grant_spaces_create:
+
+Creating and altering spaces
+****************************
+
+In the example below, ``testuser`` gets privileges allowing them to create :ref:`spaces <index-box_space>`:
+
+..  code-block:: lua
+
+    box.schema.user.grant('testuser','create','space')
+    box.schema.user.grant('testuser','write', 'space', '_schema')
+    box.schema.user.grant('testuser','write', 'space', '_space')
+
+To allow ``testuser`` to drop a space that has associated objects, add the following privileges:
+
+..  code-block:: lua
+
+    box.schema.user.grant('testuser','create,drop','space')
+    box.schema.user.grant('testuser','write','space','_schema')
+    box.schema.user.grant('testuser','write','space','_space')
+    box.schema.user.grant('testuser','write','space','_space_sequence')
+    box.schema.user.grant('testuser','read','space','_trigger')
+    box.schema.user.grant('testuser','read','space','_fk_constraint')
+    box.schema.user.grant('testuser','read','space','_ck_constraint')
+    box.schema.user.grant('testuser','read','space','_func_index')
+
+
+.. _access_control_grant_spaces_indexes:
+
+Creating and altering indexes
+*****************************
+
+In the example below, ``testuser`` gets privileges allowing them to create :ref:`indexes <index-box_index>` in the 'writers' space:
```

Comment:
The space name `writers` isn't highlighted as code, and there are more such names below (books, id_seq, sum). Please recheck the whole page.